Doc/library/bdb.rst

```diff
@@ -293,7 +293,7 @@ The :mod:`bdb` module also defines two classes:
 
```

Comment:
You can also add
```rst
   .. index:: single: quitting (bdb.Bdb attribute)
```

Reply:
Took a bit of messing around, but I think I got that in there. Please take a look.